tests/e2e_regression/run.sh

```diff
@@ -52,6 +52,9 @@ testCases=(
   'tx_volume                /v1/consensus/stats/tx_volume'
   'bucket_size              /v1/consensus/stats/tx_volume?bucket_size_seconds=300'
   'nonstandard_bucket_size  /v1/consensus/stats/tx_volume?bucket_size_seconds=301'
+  'active_accounts          /v1/consensus/stats/active_accounts'
+  'active_accounts_window   /v1/consensus/stats/active_accounts?window_step_seconds=300'
+  'active_accounts          /v1/consensus/stats/active_accounts'
```

Comment:
Nit: duplicate.

FYI, we're not using this test in any automated way yet, but we expect to.

Reply:
Right, this was supposed to be a query for `emerald` :-) Will update, thanks!